### ISSUE TITLE
Sample for n seconds

### ISSUE THREAD

**Issue by synapticarbors** [P403n1x87/austin#53]
It's not clear if there is currently a way to do this, but if I haven't missed something, it would be nice to have an argument that specified that you wanted to sample for `n` seconds. The use case that I'm thinking of is that I have a process that's running over a long period of time and I want to attach to it for some short period, sample, and then stop sampling and then pipe this to flamegraph:

```
austin -p 2712 -s | ./flamegraph.pl  > profile.svg &
```

Pyflame and py-spy hav a flag for this, which I found to be quite useful.

**Comment by P403n1x87** [comment]
Austin doesn't have an option for this currently but could be added 😄. In the meantime, I guess you can achieve what you want with some bash galore 😃 

~~~ bash
austin -p 2712 -s | ./flamegraph.pl  > profile.svg & ; sleep n ; pkill -SIGINT austin
~~~

**Comment by P403n1x87** [comment]
Fixed by #56
